storage-proofs/src/stacked/proof.rs

```diff
@@ -386,7 +402,26 @@ impl<'a, H: 'static + Hasher, G: 'static + Hasher> StackedDrg<'a, H, G> {
             }
 
             // Write the result to disk to avoid keeping it in memory all the time.
-            labels.push(DiskStore::new_from_slice(layer_size, &layer_labels)?);
+            let layer_config = StoreConfig::from_config(
+                &config,
+                format!("replica-{:?}-layer-{}", replica_id, layer),
```

Comment:
no need to use `replica_id` here either, should be part of the enum as well, sth like `CacheKey::LabelLayer(usize)`